src/redux/configureStore.js

```diff
@@ -0,0 +1,18 @@
+import { createStore, applyMiddleware, combineReducers } from 'redux';
+import createLogger from 'redux-logger';
+import booksReducer from './books/books';
+import categoriesReducer from './categories/categories';
+
+// initialize logger
+const loggerMiddleware = createLogger();
+
+// apply logger to redux
+const createStoreWithMiddleware = applyMiddleware(loggerMiddleware)(createStore);
+
+const reducer = combineReducers({
+  booksReducer,
+  categoriesReducer,
+});
+
+const configureStore = (initialState) => createStoreWithMiddleware(reducer, initialState);
+export default configureStore;
```

Comment:
- [ ] I highly recommend importing directly the `logger from redux-logger` because I give us the default options. We can only use the `createLogger` if we want to create our custom option which is not necessary for now.

       import logger from 'redux-logger'
       import { createStore, applyMiddleware, combineReducers } from 'redux';
       import booksReducer from './books/books';
      import categoriesReducer from './categories/categories';

      const reducer = combineReducers({
       booksReducer,
       categoriesReducer,
      });

       const store = createStore(
       reducer,
       applyMiddleware(logger)
       )